### DISCUSSION TITLE
r/i-booked-an-rv

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#27] [Announcements]
# r/i-booked-an-rv

终于，我定了一台房车

https://blog.ops-coffee.cn/r/i-booked-an-rv

**Comment by yangliu0109** [comment]
有错别字。
原文：例如车上做法、野外露宿等等
更改：例如车上做饭、野外露宿等等

**Reply by with122** [reply]
在车上做法事哈哈哈
